test_map.ipynb

```diff
@@ -0,0 +1,94 @@
+{
```

Comment:
Test files should be located in `tests` not in the project root directory. Or was this an accidental inclusion?

Reply:
test_map was not designed to be an actual test, but rather my own playground, i had to included it to be able to work on it from home.